pom.xml

```diff
@@ -252,6 +252,17 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>staged</id>
+      <name>staged-releases</name>
+      <url>https://repository.apache.org/content/repositories/staging/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
```

Comment:
Do we need snapshots enabled? Ideally we wouldn't by default let them be found.

Reply:
Will remove this repository once Hive 2.3.6 released.

Reply:
OK. If you end up committing this change before that, I'd leave a TODO in here to remove it.